velociraptor/autoplotter/compare.py

```diff
@@ -164,15 +164,26 @@ def recreate_single_figure(
     for line_type in valid_line_types:
         line = getattr(plot, f"{line_type}_line", None)
         if line is not None:
-            for name, data in line_data.items():
+            for color, (name, data) in enumerate(line_data.items()):
+                color = f"C{color}"
+
+                try:
+                    this_line_dict = data[plot.filename]["lines"][line_type]
+                    centers = unyt.unyt_array(
+                        this_line_dict["centers"], units=plot.x_units
+                    )
+                    heights = unyt.unyt_array(
+                        this_line_dict["values"], units=plot.y_units
+                    )
+                    errors = unyt.unyt_array(
+                        this_line_dict["scatter"], units=plot.y_units
+                    )
+                except KeyError:
```

Comment:
I do not think there will actually be a **KeyError** if the data is missing. Instead, the auto plotter will find an empty list whose units are not specified.